_data/integrations.yml

```diff
@@ -0,0 +1,632 @@
+
+    - language-weaver:
+        custom: true
+        offline_glossary: true
+
+    - sundamt
+
+    - systran:
+        custom: true
+
+    - tauyou
+
+    - tencent
+
+    - tilde:
+        custom: true
+
+    - ubiqus:
+        glossary: true
+
+    - yandex
+
+  engine_aggregator_integrations:
+    - crosslang
+
+  quality_estimation:
+    - memsource-mtqe
+
+  routing: true
+
+- id: omegat
+  name: OmegaT
+  url: https://www.omegat.org
+  type: [ cat ]
+  open_source: true
+  documentation_url: https://omegat.org/documentation
+  engine_integrations:
+    - apertium
+
+    - deepl
+
+    - google
+
+    - watson
+
+    - microsoft
+
+    - moses
+
+    - yandex
+
+  engine_plugins:
+    - tencent:
+        link: https://github.com/yoyicue/omegat-tencent-plugin
+
+  local_engine_support: true
+  fuzzy_repair: true
+  fuzzy_repair_plugin: https://github.com/transducens/OmegaT-FuzzyMatchRepair-Plugin
+  multiple_engine_suggestions: true
+
+- id: passolo
+  name: Passolo
+  url: https://www.trados.com/products/passolo/
+  type: [ cat ]
+  documentation_url: https://docs.rws.com/914743/262054/passolo-help/welcome-to-the-------------passolo-documentation
+  engine_integrations:
+    - google:
+        custom: true
+
+    - language-weaver:
+        custom: true
+
+    - microsoft:
+        custom: true
+
+  engine_plugins:
+    - deepl:
+        link: https://appstore.rws.com/language/app/deepl-mt-provider-for-passolo/1088/
+
+  multiple_engine_suggestions: true
+
+- id: smartcat
+  name: Smartcat
+  url: https://www.smartcat.ai
+  type: [ tms, cat ]
+  documentation_url: https://help.smartcat.com/hc/en-us/
+  engine_integrations:
+    - amazon:
+        custom: true
+
+    - baidu
+
+    - deepl
+
+    - google:
+        custom: true
+        glossary: true
+
+    - microsoft:
+        custom: true
+
+    - modernmt:
+        custom: true
+
+    - yandex
+
+    engine_plugins:
+    - globalese:
+        custom: true
+
+    engine_aggregator_integrations:
+    - intento:
+        glossary: true
+
+  multiple_engine_suggestions: true
+
+- id: smartling
+  name: Smartling
+  url: https://www.smartling.com
+  type: [ tms, cat ]
+  documentation_url: https://help.smartling.com/hc/en-us
+  engine_integrations:
+    - amazon
+
+    - deepl
+
+    - google:
+        custom: true
+        glossary: true
+        link: https://help.smartling.com/hc/en-us/articles/1260805168070-Translating-with-Google-AutoML
+
+    - microsoft:
+        custom: true
+
+    - promt
+
+    - systran
+
+    - watson:
+        custom: true
+
+  engine_aggregator_integrations:
+    - intento
+
+  local_engine_support: true
+  routing: true
+
+- id: trados
+  name: Trados Studio
+  url: https://www.trados.com/products/trados-studio
+  type: [ cat ]
+  documentation_url: https://docs.rws.com/980998/261377/trados-studio-2022/welcome-to-the-------------trados-studio-2022-documentation
+  engine_integrations:
+    - language-weaver:
+        custom: true
+
+  engine_plugins:
```

Comment:
Trados can also use Google and Microsoft MTs - they are hidden in this plugin: https://appstore.rws.com/language/app/mt-enhanced-plugin-for-trados-studio/604/